vendor/xclannad/wavfile.cc

```diff
@@ -738,7 +742,7 @@ void MP3FILE::Seek(int count) {
 	} while(pimpl->status != MP3FILE_impl::DONE && pimpl->write_pointer < pimpl->write_data_len);
 	return;
 }
-#elif USE_SMPEG
```

Comment:
I'm surprised this builds.

SMPEG was never really used in rlvm. This code here comes from xclannad, which had a support for several ogg and mp3 decoder libraries. The code that you're enabling has never been tested to work with rlvm. I'd rather you dropped the changes here that enable smpeg.